### PR TITLE
update computing page with components

### PR DESCRIPTION
**Pull request by joegibb**

### CODE REVIEW

**Review by Sarah-DfE — COMMENTED** [review]
Hiya @joegibb, a few little bits for you to look at.

Looks like we're missing the 'Shape your pupils' future' bit. Once you have this bit in, you can move Chris's quote to this section, rather than at the bottom of the curriculum bit.

Just looking at the chemistry page and the title is Start your journey into teaching chemistry', so think we just need to add 'computing' to the end of this H2

Also, not sure we need the three bullets right at the top as this info is covered in the curriculum

File: app/views/content/shared/teaching/_funding_support.html.erb [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <div class="funding-support">
   <p>Even if you've had financial support for a previous course, you could still be eligible for funding to support your teacher training.</p>
-  <p>You may also be able to get <a href="/funding-and-support">extra funding and support</a> if you're:</p>
+  <p>You may also be able to get extra funding and support if you're:</p>
   <ul>
     <li>a parent or carer</li>
     <li>disabled</li>
     <li>a veteran</li>
   </ul>
+  <p>Find out how you can <a href="/funding-and-support">Fund your training </a>.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/shared/teaching/_funding_support.html.erb:9]:
```suggestion
  <p>Find out how you can <a href="/funding-and-support">fund your teacher training</a>.</p>
```

File: app/views/content/life-as-a-teacher/explore-subjects/computing/_article.html.erb [hunk 2]
```diff
@@ -8,14 +8,12 @@
...
+    <%= render 'content/shared/teaching/benefits' %>
   </section>
 
   <section class="col col-720">
-    <%= render Content::QuoteComponent.new(
-      text: "Every lesson presents an opportunity to delve into the latest advancements in the field and inspire students to become innovators. Teaching computing is about fostering a deep understanding of technology while enabling students to develop the skills and resilience to thrive in this digital age.",
-      name: "Shiulee, Curriculum lead for computing and business",
-      large: true
-    ) %>
+    <!-- Add chris quote  -->
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/explore-subjects/computing/_article.html.erb:15]: I'm not sure if you need these comments really - however, if you prefer to keep them (which is fine!), this one needs updating!

```suggestion
    <!-- Add shiulee quote  -->
```

**Comment by gemmadallmandfe** [comment]
> Hiya @joegibb, a few little bits for you to look at.
> 
> Looks like we're missing the 'Shape your pupils' future' bit. Once you have this bit in, you can move Chris's quote to this section, rather than at the bottom of the curriculum bit.
> 
> Just looking at the chemistry page and the title is Start your journey into teaching chemistry', so think we just need to add 'computing' to the end of this H2
> 
> Also, not sure we need the three bullets right at the top as this info is covered in the curriculum


Agree that those three bullets can be removed - you can tell these subject pages were written by different people at different times!

Re the H2 - we've missed this on the maths page (but it is on chemistry and biology) - could you add it into the maths page as well @joegibb for consistency, either in this PR, the next one or a quick separate one?

File: app/views/content/life-as-a-teacher/explore-subjects/computing/_article.html.erb [hunk 3]
```diff
@@ -62,57 +60,26 @@
...
-    <p>Find out more about what <a href="/train-to-be-a-teacher/qualifications-you-need-to-teach">qualifications you need to train to teach</a>.</p>
-    <%= render Content::ExpanderComponent.new(
-      title: "check your qualifications",
-      text: "If you have qualifications from outside the UK, you'll need to show that they meet the standards set for teacher training in England.",
-      link_title: "You can get help comparing English and international qualifications.",
-      link_url: "/non-uk-teachers/non-uk-qualifications",
-    ) %>
+    <h2 class="heading--box-blue">Start your journey into teaching computing</h2>
+    <%= render 'content/shared/qualifications-training/check_qualifcations' %>
+     <p> Computing teacher training courses have had successful applications from candidates with a range of degrees, including business and management, data science, history and economics.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/life-as-a-teacher/explore-subjects/computing/_article.html.erb:66]:
```suggestion
     <p>Computing teacher training courses have had successful applications from candidates with a range of degrees, including business and management, data science, history and economics.</p>
```

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Review by Sarah-DfE — APPROVED** [review]
Looks good to me

**Pull request merged**